lib/chai/utils/addChainableMethod.js

```diff
@@ -58,7 +58,7 @@ var call  = Function.prototype.call,
  * @api public
  */
 
-module.exports = function (ctx, name, method, chainingBehavior) {
+module.exports = function addChainableMethod(ctx, name, method, chainingBehavior) {
```

Comment:
I'm a big fan of naming every function even if they're being assigned to a variable.
Bonus style points for this excellent practice.